src/monster.h

```diff
@@ -407,7 +407,7 @@ class monster : public Creature, public JsonSerializer, public JsonDeserializer
 
     private:
         int hp;
-        std::unordered_map<std::string, mon_special_attack> special_attacks;
+        std::map<std::string, mon_special_attack> special_attacks;
```

Comment:
That's generally the case, and I'd also say that std::map is a bit more robust and space efficient in general, so it makes for a better default.